### PR TITLE
Phone Number Validation Updated

### PR DESCRIPTION
**Pull request by saramissak**
In my js/script.js file, I rewrote and created a better phone number validation that allows for country codes, parentheses, and dashes. The contact.html form has been edited so that onsubmit and onblur the validation occurs.

### CODE REVIEW

File: portfolio/src/main/webapp/js/script.js [hunk 1]
```diff
@@ -31,12 +31,17 @@ function addRandomFact(id) {
...
-        alert("Phone  number is not valid or not valid format.");
-        return false;
-    }
+function validatePhoneNumber() {
+  var phone = $("#telephone").val();
+  var phoneNum = /^[+]?(1\-|1\s|1|\d{3}\-|\d{3}\s|)?((\(\d{3}\))|\d{3})(\-|\s)?(\d{3})(\-|\s)?(\d{4})$/g;
+  if(phone.match(phoneNum))
+  {
+    document.getElementById('invalid').innerHTML = "<p style='color:gray'> Phone Number</p>";
+	return true;
```
Comment (shenqiuz) [inline portfolio/src/main/webapp/js/script.js]: The indent space is different between Line 39 and Line 40. Line 40 should be move to left for two spaces.

File: portfolio/src/main/webapp/js/script.js [hunk 2]
```diff
@@ -31,12 +31,17 @@ function addRandomFact(id) {
...
+  }
+  else
+  {	   
+    document.getElementById('invalid').innerHTML = "<p style='color:red'> Phone Number</p>";
+	return false;
```
Comment (shenqiuz) [inline portfolio/src/main/webapp/js/script.js]: Move Line 45 to left by two spaces.

File: portfolio/src/main/webapp/js/script.js [hunk 3]
```diff
@@ -31,12 +31,17 @@ function addRandomFact(id) {
...
-        return false;
-    }
+function validatePhoneNumber() {
+  var phone = $("#telephone").val();
+  var phoneNum = /^[+]?(1\-|1\s|1|\d{3}\-|\d{3}\s|)?((\(\d{3}\))|\d{3})(\-|\s)?(\d{3})(\-|\s)?(\d{4})$/g;
+  if(phone.match(phoneNum))
+  {
+    document.getElementById('invalid').innerHTML = "<p style='color:gray'> Phone Number</p>";
+	return true;
+  }
```
Comment (shenqiuz) [inline portfolio/src/main/webapp/js/script.js]: I think the following format is more common.

if (condition) {
  ...
} else {
  ...
}

File: portfolio/src/main/webapp/js/script.js (hunk 2 above)
Comment (saramissak) [reply portfolio/src/main/webapp/js/script.js]: I don't know why it is appearing like that on github when locally they are the same, but I deleted it and redid the indentation locally. Hopefully this fixes the issue.

**Review by shabbirahussain — APPROVED** [review]

**Pull request merged**